### PR TITLE
Feature/button mixin

### PR DESCRIPTION
**Pull request by JSerhii**

### CODE REVIEW

File: docs/buttons.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ _Mixin which helps to create button with specified border and background color w
 - `$active-border` - defines button's border color when active (**default `darken($border, 12.5%)`**)
 
 ### Usage: 
-Assign primary color (variable) as button's text and background color and primary dark (variable) as border of button 
+Assign primary `$color-red` as background color and `$color-red-dark` color `$color-red-dark` as border of button
```
Comment (Nerdy-Girl) [inline docs/buttons.md]: Assign `$color-red` as background color and `$color-red-dark` as border color of button.

File: docs/buttons.md [hunk 2]
```diff
@@ -57,12 +57,11 @@ _Mixin which helps to create button with specified border and background color w
 - `$active-border` - defines button's border when active (**default `$color`**)
 
 ### Usage: 
-Assigned button with $color-primary(variable) color and $color-primary-light(variable) color of border and background
- while active.
+Assigned button with `$color-red` color and `$color-red-dark` color of border while active.
```
Comment (Nerdy-Girl) [inline docs/buttons.md]: Assign `$color-red` as button's text and border color and `$color-red-dark` as border color while active.

**Review by Nerdy-Girl — CHANGES_REQUESTED** [review]

**Comment by Nerdy-Girl** [comment]
@Serhii-Hulko Also please update changelog

**Comment by CodeMeNatalie** [comment]
@Serhii-Hulko Also please update changelog

**Review by Nerdy-Girl — APPROVED** [review]

**Pull request merged**